governance/xc_admin/packages/xc_admin_frontend/pages/index.tsx

```diff
@@ -10,6 +11,22 @@ import { MultisigContextProvider } from '../contexts/MultisigContext'
 import { PythContextProvider } from '../contexts/PythContext'
 import { classNames } from '../utils/classNames'
 
+export const getStaticProps: GetStaticProps = async () => {
+  const publisherKeyToNameMapping = fs.existsSync('publishers.json')
+    ? JSON.parse((await fs.promises.readFile('publishers.json')).toString())
```

Comment:
I couldn't help but notice that this mapping is the opposite of reference_data : (ref_data is `name -> key` and this is `key-> name` )

Reply:
The map that will be fed is key -> name